packages/ckeditor5-utils/src/inserttopriorityarray.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+import priorities from './priorities';
+
+/**
+ * @module utils/inserttopriorityarray
+ */
+
+/**
+ * The priority object descriptor.
+ *
+ *		const objectWithPriority = {
+ *			priority: 'high'
+ *		}
+ *
+ * @typedef {Object} module:utils/priorities~ObjectWithPriority
+ *
+ * @property {module:utils/priorities~PriorityString} priority Priority of the object
+ */
+
+/**
+ * Inserts any object with priority at correct index by priority so registered objects are always sorted from highest to lowest priority
```

Comment:
```suggestion
 * Inserts any object with priority at correct index by priority so registered objects are always sorted from highest to lowest priority.
```

Reply:
This is not an "any object", it must conform to the specific type with `priority` field `module:utils/priorities~ObjectWithPriority`.